workflow/scripts/convert_fusions_to_vcf.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# this script is part of arriba (https://github.com/suhrig/arriba) and published under the MIT Expat license.
+# source: https://github.com/suhrig/arriba/blob/master/scripts/convert_fusions_to_vcf.sh
```

Comment:
Please use the commit you have taken this from instead of master branch in the URL..